pkg/pipelinerunmetrics/metrics.go

```diff
@@ -171,33 +191,61 @@ func viewRegister(cfg *config.Metrics) error {
 		Aggregation: view.Count(),
 		TagKeys:     prCountViewTags,
 	}
+	prTotalView = &view.View{
+		Description: prTotal.Description(),
+		Measure:     prTotal,
+		Aggregation: view.Count(),
+		TagKeys:     []tag.Key{statusTag},
+	}
+
 	runningPRsCountView = &view.View{
 		Description: runningPRsCount.Description(),
 		Measure:     runningPRsCount,
 		Aggregation: view.LastValue(),
 	}
+	runningPRsView = &view.View{
+		Description: runningPRs.Description(),
+		Measure:     runningPRs,
+		Aggregation: view.LastValue(),
+	}
+
 	runningPRsWaitingOnPipelineResolutionCountView = &view.View{
 		Description: runningPRsWaitingOnPipelineResolutionCount.Description(),
 		Measure:     runningPRsWaitingOnPipelineResolutionCount,
 		Aggregation: view.LastValue(),
 	}
+	runningPRsWaitingOnPipelineResolutionView = &view.View{
+		Description: runningPRsWaitingOnPipelineResolution.Description(),
+		Measure:     runningPRsWaitingOnPipelineResolution,
+		Aggregation: view.LastValue(),
+	}
+
 	runningPRsWaitingOnTaskResolutionCountView = &view.View{
 		Description: runningPRsWaitingOnTaskResolutionCount.Description(),
 		Measure:     runningPRsWaitingOnTaskResolutionCount,
 		Aggregation: view.LastValue(),
 	}
+	runningPRsWaitingOnTaskResolutionView = &view.View{
+		Description: runningPRsWaitingOnTaskResolution.Description(),
+		Measure:     runningPRsWaitingOnTaskResolution,
+		Aggregation: view.LastValue(),
+	}
 
 	return view.Register(
 		prDurationView,
 		prCountView,
+		prTotalView,
 		runningPRsCountView,
+		runningPRsView,
 		runningPRsWaitingOnPipelineResolutionCountView,
+		runningPRsWaitingOnPipelineResolutionView,
 		runningPRsWaitingOnTaskResolutionCountView,
+		runningPRsWaitingOnTaskResolutionView,
 	)
 }
 
 func viewUnregister() {
-	view.Unregister(prDurationView, prCountView, runningPRsCountView, runningPRsWaitingOnPipelineResolutionCountView, runningPRsWaitingOnTaskResolutionCountView)
+	view.Unregister(prDurationView, prCountView, prTotalView, runningPRsCountView, runningPRsView, runningPRsWaitingOnPipelineResolutionCountView, runningPRsWaitingOnTaskResolutionCountView)
```

Comment:
Shouldn't the list of things we unregister the same as the one we register above ? Some of them are missing, like
`runningPRsWaitingOnPipelineResolutionView` and another one.

Reply:
Yes, we should. Let me add those.

Reply:
I should probably change the format here to a single line per argument. It will help.